gui-doc/scribblings/gui/slider-class.scrbl

```diff
@@ -45,8 +45,8 @@ If @racket[label] is a string, it is used as the label for the slider.
 The @racket[min-value] and @racket[max-value] arguments specify the
  range of the slider, inclusive. The @racket[init-value] argument
  optionally specifies the slider's initial value. If the sequence
- [@racket[min-value], @racket[initial-value], @racket[maximum-value]]
- is not increasing, @|MismatchExn|.
+ [@racket[min-value], @racket[init-value], @racket[max-value]]
+ is not non-decreasing, @|MismatchExn|.
```

Comment:
I would avoid double negation, which is confusing to me.

My proposal is to say:

```
The sequence [@racket[min-value], @racket[init-value], @racket[max-value]] must be non-decreasing. Otherwise, @|MismatchExn|
```

Reply:
Sure, I approve of this change.  I had thought it was a little awkward at the time but I didn't have a great idea (except possibly to just write out the invariant `(<= min-value init-value max-value)`).